csharp/src/Google.Protobuf/Google.Protobuf.nuspec

```diff
@@ -5,20 +5,34 @@
     <title>Google Protocol Buffers C#</title>
     <summary>C# runtime library for Protocol Buffers - Google's data interchange format.</summary>
     <description>See project site for more info.</description>
-    <version>3.0.0-beta*</version>
+    <version>3.0.0-alpha4</version>
```

Comment:
0.x should always be treated as a prerelease, as far as I'm aware.

While the GRPC code requires a specific alpha or beta release, I'd stick to keeping that specificity in the dependencies. Once we've released 3.0.0, you can just depend on ">= 3.0.0 < 4.0"